src/tools/rbd_mirror/ImageReplayer.h

```diff
@@ -195,6 +196,35 @@ class ImageReplayer {
     ImageReplayer<ImageCtxT> *replayer;
   };
 
+  struct ReplayStatusMaker {
```

Comment:
Super nitpicks: (1) it might be nice to break this out into the "image_replayer" namespace since it is small, doesn't need to depend on accessing ImageReplayer private state (at least that I could see), and could allow unit testing (in the future) -- (2) the "Maker" suffix doesn't sit right with me for some reason, perhaps "Generator" or "Formatter"?